branch.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+    ALTER TABLE participants DROP CONSTRAINT participants_api_key_key;
```

Comment:
What's up with this? What was this constraint doing and why don't we need it anymore?